### DISCUSSION TITLE
Not working forme... maybe?

### DISCUSSION THREAD

**Discussion by wckkyuubi** [trickerer01/Ruxx#2] [Q&A] [answered]
Hi. ruxx.exe not working for me, maybe is not compatible with 32bits?

**Comment by trickerer01** [comment, marked as answer]
The last 32-bit build is [1.0.118](https://github.com/Trickerer01/Ruxx/releases/tag/2021.02.15).

Didn't expect to see anyone use 32bit system in 2021.

**Reply by wckkyuubi** [reply]
:C oh well thanks. I'm going to migrate to x64 soon

**Reply by trickerer01** [reply]
Here is a 32-bit build for you: [1.0.124](https://github.com/Trickerer01/Ruxx/releases/tag/2021.04.13)

**Reply by wckkyuubi** [reply]
thanks dude <3
